tests/Integration/Config/ConfigFileFinderTest.php

```diff
@@ -11,7 +11,7 @@ public function test_config_loader_throw_when_it_cant_find_a_psh_file()
     {
         $loader = new ConfigFileFinder();
 
-        $this->expectException(\RuntimeException::class);
+        $this->setExpectedException(\RuntimeException::class);
```

Comment:
Out of curiosity, why should't that syntax work with humbug?
I have used it in my projects like https://github.com/bcremer/LineReader/blob/master/tests/LineReaderTest.php#L46 and it just works fine with it.